taca/storage/storage.py

```diff
@@ -250,7 +255,8 @@ def _send_to_swestore(file_path, dest, remove=True):
             logger.info('Run {} successfully compressed! Removing from disk...'
                         .format(run))
             shutil.rmtree(run)
-            _send_to_swestore('{}.tar.bz2'.format(run), CONFIG.get('storage').get('irods').get('irodsHome'))
+            if not compress_only:
+                _send_to_swestore('{}.tar.bz2'.format(run), CONFIG.get('storage').get('irods').get('irodsHome'))
         else:
             logger.info("Run {} is not {} days old yet. Not archiving"
                         .format(run, str(days)))
```

Comment:
damn it! I choose the wrong commit when merging robin's last PR... very well spotted, I'll fix it right now